Gemfile.lock

```diff
@@ -0,0 +1,270 @@
+GEM
```

Comment:
Gemfile.lock should be in your .gitignore so it should not get checked into GitHub.